visidata/wrappers.py

```diff
@@ -10,7 +10,10 @@
 option('null_value', None, 'a value to be counted as null', replay=True)
 
 def isNullFunc():
-    return lambda v,nulls=set([None, options.null_value]): v in nulls or isinstance(v, TypedWrapper)
+    return lambda v,nulls=set([None, options.null_value]): any((
+        *(v == n for n in nulls),
```

Comment:
Handling it this way, we don't need to see if a non-hashable column value (like a list) is in the `nulls` set.

Reply:
We need to be really careful when modifying isNullFunc; if the returned function is even a little slower, this will make several operations (save, freq, describe, etc) significantly slower for larger datasets. So we'd have to measure this change carefully to see how it performs.  By inspection though I think it's definitively slower, as the isinstance() passed to any() is calculated every time, whereas in the original there is an "early out" if the v value is already null.

Maybe this should be `lambda v, nullv=options.null_value: v is None or v == nullv or isinstance(v, TypedWrapper)` with a special case of `lambda v: v is None or isinstance(v, TypedWrapper)` for when nullv is just None.  We'd have to measure to see if there's a performance penalty for this.  I agree though that it would be best for null checks on lists/dicts/objects to not raise an exception.

Reply:
Yikes you're right, I did some testing with moderately sized files and a describe was ~5 times slower. I updated it based on your feedback and it looks like the performance is back to what it was before, but I'll have to give it a bit more of a workout to be sure.

Reply:
With some local benchmarks on a bigger/slower dataset, I was seeing timings of just over 7 seconds to describe a sheet with my "slow" change. That same describe reliably took ~5 seconds before this PR, and the same with your suggestions. Happy to test anything else, but from what I can tell the performance looks good now.